native/NativeSampleApps/MobileSyncExplorer/src/com/salesforce/samples/mobilesyncexplorer/ui/MainActivity.java

```diff
@@ -75,7 +82,7 @@
  * @author bhariharan
  */
 public class MainActivity extends SalesforceListActivity implements
-		OnQueryTextListener, OnCloseListener,
```

Comment:
Note `SearchView` has a long-standing bug with its `OnCloseListener` not being called in Jetpack.  There might be other ways, but look below for a `OnActionExpandListener` which is one work-around.